data/Makefile.am

```diff
@@ -31,21 +31,23 @@ dictsdir = $(pkgdatadir)/dicts
 # rf. i18n appdata.xml
 # http://www.freedesktop.org/software/appstream/docs/sect-Quickstart-Translation.html
 # https://blogs.gnome.org/hughsie/2016/01/25/appdata-and-the-gettext-domain/
-appdata_in_files = ibus-anthy.appdata.xml.in
-appdata_DATA = $(appdata_in_files:.xml.in=.xml)
-appdatadir=$(datadir)/appdata
-@INTLTOOL_XML_RULE@
+appstream_in_files = org.freedesktop.ibus.engine.anthy.metainfo.xml.in
```

Comment:
I was trying to better match the [AppStream specification](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-InputMethod.html) here (ie. `id` should follow the standard reverse-domain-name scheme) and what other ibus modules like `ibus-hangul` seems to be doing for consistency. While I generally agree with you about git history for source code, I don't think it matters much for data files like this as changes are usually straightforward. But of course that's up to you.

Reply:
Very sorry, could you rebase your repo again?
I just now noticed you created a patch for github.com/ibus/anthy repo but this repo is used only when I release a tarball and the devel repo is https://github.com/fujiwarat/ibus-anthy/ because I sometimes commit changes forcibly.
I updated github.com/ibus/anthy repo now.

Since I have already maintain the file as  ibus-anthy.appdata.xml.in,  I'd keep the file name and I'm still not sure if the name `org.freedesktop.ibus.engine.anthy.metainfo.xml` won't be changed in the future.
E.g. we have the experience to rename appdata.xml to metainfo.xml. I may change the filename to the title format likes `org.freedesktop.IBus.Engine...`.
I'd think the source filename keeps a short length, `ibus-anthy.appdata.xml.in`, and managed in git and the destination filename is not maintained in git and can be changed in case.

However if you still suggest to rename ibus-anthy.appdata.xml.in, I will follow you.

Reply:
Oh I'm sorry I had no idea that this is not the development repository. Maybe you could add a note in the GitHub description to make it clear. Do you want me re-open this PR against `fujiwarat/ibus-anthy`?

Anyway, I've rebased the patch: it now respects the logic implemented upstream: sticking with `ibus-anthy.appdata.xml.in` in the repo. and generating `org.freedesktop.ibus.engine.anthy.metainfo.xml`, which is totally fine by me.

Reply:
Thank you. I may confuse people a little with a repository. I expect to file issues and pull requests for ibus/ibus-anthy but the devel repo is fujiwarat/ibus-anthy so that the issue IDs in each git subject can be linked to the ones in ibus/ibus-anthy when I integrate commits in fujiwarat/ibus-anthy to ibus/ibus-anthy.
I updated a note in ibus/ibus-anthy now but it could be still a too short notice..